src/inversify.config.ts

```diff
@@ -28,6 +28,7 @@ export function initContainer (): Container {
 
     // Env
     container.bind<string>(TYPES.CDN_HOST).toConstantValue(getEnvOrThrow("CDN_HOST"))
+    container.bind<boolean>(TYPES.CDN_HOST_LONG_PREFIX).toConstantValue(Boolean(getEnvOrThrow("CDN_HOST_LONG_PREFIX")))
```

Comment:
Could we use getEnvOrDefault instead so that it defaults to false, meaning that environments not yet updated do not need config changes?

Reply:
As discussed with @nevans4-ch this and other chamges are currently on hold